### PR TITLE
404 mobile first layout

### PR DESCRIPTION
**Pull request by nutsil**
Initial layout styles following the design provided on the ticket.

### Screenshots
Screen Shot 2020-10-27 at 20 32 12

### CODE REVIEW

File: src/components/App/App.component.js [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
 export function App() {
-  return <div></div>;
+  return (
+    <div className="app-wrapper">
+      <header className="header">
+        <TitlePage />
+      </header>
+      {/* Remover name of classes for something more meaningful */}
+      <main className="area-2">
+        <div className="area-2-a">
```
Comment (nicolasjuarezn) [inline src/components/App/App.component.js:15]: Minor:

Please remember to remove the comments on the code, and to apply meaningful names to the classes

File: src/components/App/App.component.js [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+            width="286"
+            height="238"
+            className="scarecrow-image"
+          />
+        </div>
+        <div className="area-2-b">
+          <ContentBody />
+        </div>
+      </main>
+      <footer className="area-3"></footer>
```
Comment (nicolasjuarezn) [inline src/components/App/App.component.js:28]: Minor:
Same comment as above.

**Review by nicolasjuarezn — CHANGES_REQUESTED** [review]
Minor changes were requested

File: src/components/App/App.component.js (hunk 1 above)
Comment (nutsil) [reply src/components/App/App.component.js]: Done!

File: src/components/App/App.component.js (hunk 2 above)
Comment (nutsil) [reply src/components/App/App.component.js:27]: Done!

**Review by nicolasjuarezn — APPROVED** [review]
LGTM

**Pull request merged**